curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98fb.md

```diff
@@ -7,18 +7,18 @@ dashedName: step-51
 
 # --description--
 
-You can make an instant color change in a gradient by giving the transition zero space like this:
+Give `transition` a value to create an image of your **variables' colors**, it will look like this:
```

Comment:
This feels more confusing to me. What about this:
`Gradient transitions often gradually change from one color to another. You can make the change a solid line like this:`

Reply:
sure, I was tunnel visioned. on variables colors.

With this, it's good reminder for old challenges. and more clear. I will implement it